fast-logger/System/Log/FastLogger.hs

```diff
@@ -202,7 +202,7 @@ newFastLogger typ = case typ of
     LogFile fspec bsize -> rotateLoggerInit fspec bsize
     LogCallback cb flush -> return (\ str -> cb str >> flush, noOp )
   where
-    stdLoggerInit lgrset = return (pushLogStr lgrset, noOp)
+    stdLoggerInit lgrset = return (pushLogStr lgrset, rmLoggerSet lgrset)
```

Comment:
Sorry i mix this commit up, @kazu-yamamoto I just realize we need to flush stdio too on clean up, do you think this change is necessary?